repose-aggregator/src/docs/asciidoc/filters/index.adoc

```diff
@@ -2,15 +2,104 @@
 
 include::../_includes/in-progress.adoc[]
 
-<<add-header.adoc#,Add Header Filter>>::
-    Enables users to add new headers and header values to a request and/or response.
+== <<add-header.adoc#,Add Header Filter>>
```

Comment:
Asciidoctor logs a warning if a lower-level header is used without having a parent header exactly one level above it.

So in this case, the page title is header level 1 and the filters are currently header level 2. Making the filters header level 3 causes Asciidoctor to complain:
> asciidoctor: WARNING: index.adoc: line 5: section title out of sequence: expected level 1, got level 2